### PR TITLE
OSDOCS-1694 Add mutable Ingress Controller endpoint publishing strategy release note

### PR DESCRIPTION
**Pull request by bmcelvee**
https://issues.redhat.com/browse/OSDOCS-1694

Relates to https://github.com/openshift/openshift-docs/pull/39451.

Preview: https://deploy-preview-41323--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes#ocp-4-10-ingress-controller-endpoint-publishing-strategies

### CODE REVIEW

**Comment by bmcelvee** [comment]
@Miciah PTAL, thanks! This is to add the release note.

**Review by adellape — APPROVED** [review]

**Comment by bmcelvee** [comment]
@Miciah would you mind reviewing this release note, please? Thanks!

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -565,6 +565,13 @@ You can now create DNS records by using the Red Hat External DNS Operator on clo
 
 For more information, see xref:../networking/external_dns_operator/nw-installing-external-dns-operator.adoc#nw-installing-external-dns-operator[Installing the External DNS Operator].
 
+[id="ocp-4-10-ingress-controller-endpoint-publishing-strategies"]
+==== Mutable Ingress Controller endpoint publishing strategy
+
+Cluster administrators can now configure the Ingress Controller endpoint publishing strategy in {product-title}.
```
Comment (Miciah) [inline release_notes/ocp-4-10-release-notes.adoc:571]: The specific change in OpenShift 4.10 is that the load-balancer scope can be changed between "Internal" and "External".  Some other fields were already mutable, and other fields are still not mutable.  https://github.com/openshift/openshift-docs/pull/39451#discussion_r804251703 has details.  Maybe it isn't necessary to repeat all that here as you have the cross-reference, but I'd suggest that the release note specifically mention that load-balancer scope is newly mutable (as opposed to _all_ endpoint publishing strategy settings).

**Pull request merged**